packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -0,0 +1,123 @@
+        l2BlockTime = stdJson.readUint(_json, "$.l2BlockTime");
+        maxSequencerDrift = stdJson.readUint(_json, "$.maxSequencerDrift");
+        sequencerWindowSize = stdJson.readUint(_json, "$.sequencerWindowSize");
+        channelTimeout = stdJson.readUint(_json, "$.channelTimeout");
+        p2pSequencerAddress = stdJson.readAddress(_json, "$.p2pSequencerAddress");
+        batchInboxAddress = stdJson.readAddress(_json, "$.batchInboxAddress");
+        batchSenderAddress = stdJson.readAddress(_json, "$.batchSenderAddress");
+        l2OutputOracleSubmissionInterval = stdJson.readUint(_json, "$.l2OutputOracleSubmissionInterval");
+        _l2OutputOracleStartingTimestamp = stdJson.readInt(_json, "$.l2OutputOracleStartingTimestamp");
+        l2OutputOracleStartingBlockNumber = stdJson.readUint(_json, "$.l2OutputOracleStartingBlockNumber");
+        l2OutputOracleProposer = stdJson.readAddress(_json, "$.l2OutputOracleProposer");
+        l2OutputOracleChallenger = stdJson.readAddress(_json, "$.l2OutputOracleChallenger");
+        finalizationPeriodSeconds = stdJson.readUint(_json, "$.finalizationPeriodSeconds");
+        proxyAdminOwner = stdJson.readAddress(_json, "$.proxyAdminOwner");
+        baseFeeVaultRecipient = stdJson.readAddress(_json, "$.baseFeeVaultRecipient");
+        l1FeeVaultRecipient = stdJson.readAddress(_json, "$.l1FeeVaultRecipient");
+        sequencerFeeVaultRecipient = stdJson.readAddress(_json, "$.sequencerFeeVaultRecipient");
+        governanceTokenName = stdJson.readString(_json, "$.governanceTokenName");
+        governanceTokenSymbol = stdJson.readString(_json, "$.governanceTokenSymbol");
+        governanceTokenOwner = stdJson.readAddress(_json, "$.governanceTokenOwner");
+        l2GenesisBlockGasLimit = stdJson.readUint(_json, "$.l2GenesisBlockGasLimit");
+        l2GenesisBlockBaseFeePerGas = stdJson.readUint(_json, "$.l2GenesisBlockBaseFeePerGas");
+        gasPriceOracleOverhead = stdJson.readUint(_json, "$.gasPriceOracleOverhead");
+        gasPriceOracleScalar = stdJson.readUint(_json, "$.gasPriceOracleScalar");
+        eip1559Denominator = stdJson.readUint(_json, "$.eip1559Denominator");
+        eip1559Elasticity = stdJson.readUint(_json, "$.eip1559Elasticity");
+        l2GenesisRegolithTimeOffset = stdJson.readUint(_json, "$.l2GenesisRegolithTimeOffset");
+    }
+
+    function l1StartingBlockTag() public returns (bytes32) {
+        try vm.parseJsonBytes32(_json, "$.l1StartingBlockTag") returns (bytes32 tag) {
+            return tag;
+        } catch {
+            try vm.parseJsonString(_json, "$.l1StartingBlockTag") returns (string memory tag) {
+                string[] memory cmd = new string[](3);
+                cmd[0] = "/usr/bin/bash";
+                cmd[1] = "-c";
+                cmd[2] = string.concat("cast block ", tag, " --json | jq -r .hash");
+                bytes memory res = vm.ffi(cmd);
+                return abi.decode(res, (bytes32));
```

Comment:
This is repeated - can refactor into a new function, but nbd.

Reply:
you are right, got slightly lazy here :D